1st-page/Blog.html

```diff
@@ -0,0 +1,161 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Page3 Lesson1 Assignment</title>
+</head>
+
+<style>
+
+    .header {
+        color: white;
+        background-color: #003366;
+        padding: 20px;
```

Comment:
This is fine for Assignment 2 but make sure up update css to use responsive units such as %, em, rem by lesson 3. px are no longer acceptable units (they are not responsive).

Reply:
These should really be updated to % or em by Assignment 3.